### PR TITLE
Add FemDomEmpire Scraper

### PR DESCRIPTION
**Pull request by olimeca**
Add FemDomEmpire Scraper. Need help with tags. They are available but I could not figure out how to scrape them.

### CODE REVIEW

File: scrapers/FemDomEmpire.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: "FemDomEmpire"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - femdomempire.com/tour/trailers/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="title"]/h3
```
Comment (bnkai) [inline scrapers/FemDomEmpire.yml:10]: the specific regex gets also the `More Updates` as a second result
```yaml
      Title: //div[@class="videoDetails clear"]/h3
```
seems safer

File: scrapers/FemDomEmpire.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  - action: scrapeXPath
+    url:
+      - femdomempire.com/tour/trailers/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="title"]/h3
+      Date:
+        selector: //div[@class="videoInfo clear"]/p
```
Comment (bnkai) [inline scrapers/FemDomEmpire.yml:12]: the selector matches a lot more paragraphs than the date one
```yaml
       selector: //p[span[contains(.,"Date Added")]]
```
only returns the date paragraph

File: scrapers/FemDomEmpire.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        postProcess:
+          - replace:
+            - regex: "Date Added:"
+              with:
+          - parseDate: January 2, 2006      
+      Details: //div[@class="videoDetails clear"]/p
+      Performers:
+        Name: //li[@class="update_models"]/a
+      #Tags:
```
Comment (bnkai) [inline scrapers/FemDomEmpire.yml:21]:
```yaml
      Tags:
        Name: //ul/li[@class="label" and contains(.,"Categories")]/..//a
```
seems to work

File: scrapers/FemDomEmpire.yml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        #Name:
+      Image:
+        selector: //img[@class="update_thumb thumbs stdimage"]/@src0_1x
+        postProcess:
+          - replace:
+            - regex: .*/tour/content//
+              with: "https://femdomempire.com/tour/content/"
+
```
Comment (bnkai) [inline scrapers/FemDomEmpire.yml:29]: forgot the studio
```yaml
      Studio:
        Name:
          fixed: FemDomEmpire
```

**Comment by olimeca** [comment]
Suggested edits made.

**Pull request merged**